packages/event-bus-local/src/index.ts

```diff
@@ -0,0 +1,5 @@
+import Loader from "./loaders"
+import LocalEventBus from "./services/event-bus-local"
+
+export const services = [LocalEventBus]
```

Comment:
Suggestion: I might have already suggested it but I don't recall. Would it make more sense to expect an object here? Which can be easily typed and the type can be used by the consumer to know the prop that needs to be in the object. I would argue that if we do evolve the module api it will be easier to update the object type and for the consumer to be aware of the changes. I would also argue that the autocompletion would be better and improve the dx of writing a module

Reply:
Cause with an approach of everything is exported as const at the too level do not provide any completion or i fo about what is expected or not and require for new users to go forward/backward to the doc or the code to see what is expected to be exported

Reply:
I think this is an excellent suggestion. Must have missed it the first time around.

I am thinking something along these lines, should do the job:
```ts
export type LoaderOptions = {
  container: MedusaContainer
  configModule: ConfigModule
  logger: Logger
}

type ModuleExports = {
  loaders: (options: LoaderOptions) => Promise<void>[]
  services: ClassConstructor<unknown>[]
}

// In a module

export default {
  services: []
  loaders: []
} as ModuleExports
```

Let me know what you think :)

Reply:
Sounds very good ☺️ just the loaders type might need to looks like (...)[] otherwise I think it might be typed as returning an array of promises

Reply:
Ah my mad - was too quick. Great, will add that :)

Reply:
One question, should the logger be passed as part of the options? It is resolvable from the container so I think we can remove it no?

Reply:
It is passed to all existing loaders, so I added it for consistency

Reply:
Make sense 💪 let's got with that then